### PR TITLE
Config API to provide custom brokered message converter

### PR DESCRIPTION
**Pull request by melkio**
Connects to #278 

@Particular/azure-service-bus-maintainers please review!

### CODE REVIEW

File: src/Tests/Configuration/When_configuring_custom_brokered_message_to_incoming_message_converter.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    using Microsoft.ServiceBus.Messaging;
+    using NUnit.Framework;
+    using Settings;
+
+    [TestFixture]
+    [Category("AzureServiceBus")]
+    public class When_configuring_custom_brokered_message_to_incoming_message_converter
+    {
+        [Test]
+        public void Default_value_should_e_configured()
```
Comment (SeanFeldman) [inline src/Tests/Configuration/When_configuring_custom_brokered_message_to_incoming_message_converter.cs]: "should_be"?

File: src/Tests/Configuration/When_configuring_custom_outgoing_message_to_brokered_message_converter.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    using Microsoft.ServiceBus.Messaging;
+    using NUnit.Framework;
+    using Settings;
+
+    [TestFixture]
+    [Category("AzureServiceBus")]
+    public class When_configuring_custom_outgoing_message_to_brokered_message_converter
+    {
+        [Test]
+        public void Default_value_should_e_configured()
```
Comment (SeanFeldman) [inline src/Tests/Configuration/When_configuring_custom_outgoing_message_to_brokered_message_converter.cs]: "should_be"

**Comment by yvesgoeleven** [comment]
Looks good to me

**Comment by SeanFeldman** [comment]
I'll fix the spelling, rebase, and merge.

**Pull request merged**